target/linux/ath79/dts/qca9533.dtsi

```diff
@@ -0,0 +1,236 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
```

Comment:
As I've said qca953x.dtsi is suitable for both qca9531 and qca9533. You don't need to create this file.
There are several mistakes in this file, but as it isn't needed in the first place I'm not going to point them all out :)